charts/sysdig-stackdriver-bridge/templates/deployment.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "sysdig-stackdriver-bridge.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "sysdig-stackdriver-bridge.name" . }}
+{{ include "sysdig-stackdriver-bridge.labels" . | indent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ include "sysdig-stackdriver-bridge.name" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ include "sysdig-stackdriver-bridge.name" . }}
+    spec:
+      containers:
+        - name: stackdriver-webhook-bridge
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
```

Comment:
Replace with this default: 

image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"